### PR TITLE
Add doc migrating-from-ei-7.0.0.md file

### PR DESCRIPTION
**Pull request by isuruuy429**
## Purpose
This PR contains the adding the new doc of migrating from EI 7.0.0 to EI 7.1.0.

### CODE REVIEW

File: en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Migrating from WSO2 EI 7.0.0 to WSO2 EI 7.1.0
+
+### Set up the migration
+
+-	Make a backup of the existing database used by the current EI 7.0.0 deployment. This backup is necessary in case the migration causes any issues in the existing database.
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md:5]: We didn't support databases in EI 7.0.0 did we? I think we can remove this section.
@Nirothipan - could you also confirm?

File: en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+-	Download and install EI 7.1.0 in your environment. The home directory of your installation will be referred to as `<MI_HOME>` from hereon.
+	-	[On a VM](../../../setup/installation/install_in_vm)
+	-	[On Docker](../../../setup/installation/run_in_docker)
+	-	[On Kubernetes](../../../setup/installation/run_in_kubernetes)
+-	You can [use WSO2 Update Manager](https://docs.wso2.com/display/updates/) to get the latest available updates.
+
+	!!! Info
+		Note that you need a valid [WSO2 subscription](https://wso2.com/subscription) to use updates in a production environment.
+
+### Migrating databases
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md:15]: We didn't support databases in EI 7.0.0 did we? I think we can remove this section.
@Nirothipan - could you also confirm?

File: en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+You can use the same databases that you used for the EI 7.0.0 of WSO2 with WSO2 EI 7.1.0. Simply copy the database configurations to the `MI-HOME/conf/deployment.toml` file as you did in WSO2 EI 7.0.0. 
+
+### Migrating the registry
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md:18]: Shouldn't we be copying the MI_HOME/registry/ folder?

File: en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md (hunk 1 above)
Comment (Nirothipan) [reply en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md:5]: Yes +1 . I guess this has been taken from migrating from 6xx doc.

File: en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md (hunk 3 above)
Comment (Nirothipan) [reply en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md:18]: I think better we not copy and encourage ppl to deploy capps and not to mess with registry manually

Comment (Nirothipan) [reply en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md:18]: we have to mention that in both migration docs. @sdkottegoda  any other opinion ?

File: en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md (hunk 2 above)
Comment (Nirothipan) [reply en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md:15]: +1 to make as 'Migrating user stores'  .. we can say just plug the same here and point to user store docs.

File: en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md (hunk 3 above)
Comment (nilminiwso2) [reply en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md:17]: I feel like the statement 'You can directly migrate the artifacts to the EI 7.1.0 by adding the carbon applications to the `MI-HOME/repository/deployment/server/carbonapps` folder. ' is not very clear here. We should probably explain the Carbon applications as it relates to the registry a bit more. wdyt?

Comment (Nirothipan) [reply en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md:17]: https://github.com/wso2/docs-ei/blob/master/en/micro-integrator/docs/setup/deployment/migrating-from-ei-6.x.x.md#migrating-the-registry how about a variant of msg from this

File: en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md (hunk 1 above)
Comment (isuruuy429) [reply en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md:5]: Removed.

File: en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md (hunk 2 above)
Comment (isuruuy429) [reply en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md]: Done.

**Pull request merged**